[2024-01-28T16:02:31.046+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-03T00:00:00+00:00 [queued]>
[2024-01-28T16:02:31.116+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-03T00:00:00+00:00 [queued]>
[2024-01-28T16:02:31.131+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-28T16:02:31.250+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2023-03-03 00:00:00+00:00
[2024-01-28T16:02:31.269+0000] {standard_task_runner.py:60} INFO - Started process 3488 to run task
[2024-01-28T16:02:31.284+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2023-03-03T00:00:00+00:00', '--job-id', '1378', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmpq0p18ed0']
[2024-01-28T16:02:31.313+0000] {standard_task_runner.py:88} INFO - Job 1378: Subtask get_stackoverflow_data
[2024-01-28T16:02:31.501+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-03-03T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-28T16:02:32.302+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2023-03-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-03T00:00:00+00:00'
[2024-01-28T16:02:33.033+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-28T16:02:33.094+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20230303T000000, start_date=20240128T160231, end_date=20240128T160233
[2024-01-28T16:02:33.160+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T16:02:33.214+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
